### DISCUSSION TITLE
Book4 Ch15 奇异值分解 Page 5-6

### DISCUSSION THREAD

**Discussion by RickMCCC** [Visualize-ML/Book4_Power-of-Matrix#15] [General]
作者您好，我是一个正在复习线性代数的学生，看到您的讲解后感觉很有收获。

不过我有一点小建议：在Book4 Ch15 奇异值分解 Page 5-6 这一部分的插图中，在对原空间中的基向量进行线性变换后，得到了新的一组基向量。这里可否在向量变换的过程中，将背景的灰色网格也做同样的拉伸呢（就像3B1B视频中矩阵线性变换一节中所做的那样）？因为新空间中的向量组也是彼此正交的，这一点非常重要，而现有的图片有些容易让人混淆。

**Comment by Visualize-ML** [comment]
你这个idea很好！我先记下，回头做出来！欢迎继续批评指正

Cheers,
James
